README.md

```diff
@@ -58,6 +58,7 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
 - [Centerpoint Networks Pvt. Ltd.](http://centerpoint-networks.com/) | Golfutar, Kathmandu | Software Solutions, Web Application Development, ERP, CRM
 - [Channakya Software Pvt. Ltd.](https://www.channakyasoft.com/) | Sinamangal, Kathmandu | We build banking software, assets management software, attendance management software, payroll management software, human resource management and cheque management software
 - [Chhito Connect Pvt. Ltd.](https://www.chhitoconnect.com/) | Anamnagar, Kathmandu | Digital Banking Products
+- [Chulo Solutions Pvt. Ltd.](https://chulo-solutions.com) | Kumaripati, Lalitpur | Software development company in Nepal having its parent company, Namespace Inc.
```

Comment:
@jyapujuju Thank you for contributing. As a best practice, I've tried to avoid including names of the parent company in the description. 

I instead suggest you to write a better description for your company so that people can know what your company does.

```suggestion
- [Chulo Solutions Pvt. Ltd.](https://chulo-solutions.com) | Kumaripati, Lalitpur | Software development company
```